packages/odyssey-storybook/src/guidelines/DownstreamDevelopment.stories.mdx

```diff
@@ -0,0 +1,313 @@
+
+```js
+const { default: postcssOdyssey } = require("@okta/odyssey-postcss-preset");
+
+module.exports = (ctx) => {
+  if (!ctx.odyssey) {
+    return {};
+  }
+
+  const options = Object.assign(
+    ctx.odyssey,
+    ctx.env === "production"
+      ? {
+          logical: {
+            dir: "ltr",
+            preserve: false,
+          },
+          autoprefixer: {
+            grid: "autoplace",
+            env: "production",
+          },
+        }
+      : {
+          modules: {
+            ...ctx.odyssey.modules,
+            generateScopedName: "[name]__[local]___[hash:base62:6]",
+          },
+        }
+  );
+
+  return {
+    plugins: [postcssOdyssey(options)],
+  };
+};
+```
+
+Again, you can further configure each part of this preset by passing arguments
+within your postcss configuration. You may also want to configure the
+preset differently depending on the environment. You can see an example
+of this above using the `ctx.env === "production"` expression.
+
+[7]: https://postcss.org
+
+## Your first Odyssey component
+
+With your build toolchain in place it is time to install additional
+dependencies used at run time:
+
+```sh
+  $ npm install \
+      react \
+      react-dom \
+      @okta/odyssey-react-theme \
+      @okta/odyssey-react
+```
+
+Now create a new component file using JSX and typescript: `HelloWorld.tsx`
+
+```js
+import React from "react";
+import { Box } from "@okta/odyssey-react";
+import { withTheme } from "@okta/odyssey-react-theme";
+import { theme } from "./HelloWorld.theme";
+import styles from "./HelloWorld.module.css";
+
+export const HelloWorld = withTheme(
+  theme,
+  styles
+)(() => (
+  <Box
+    as="p"
+    display="flex"
+    alignItems="center"
+    justifyContent="center"
+    className={styles.root}
+  >
+    <Box
+      as="strong"
+      className={styles.text}
+      color={false}
+      fontSize={false}
+      children="Hello World"
+    />
+  </Box>
+));
+```
+
+Then render your component within a react-dom entrypoint:
+
+```js
+import React from "react";
+import { render } from "react-dom";
+import { HelloWorld } from "./HelloWorld";
+
+const rootElement = document.getElementById("root");
+render(<HelloWorld />, rootElement);
+```
+
+Next, create a sibling CSS module file: `HelloWorld.module.css`:
+
+```css
+.root {
+  font-size: 3rem;
+  color: var(--Color);
+}
+```
+
+And finally, create a sibling theme module file: `HelloWorld.theme.ts`:
+
+```js
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  Color: theme.ColorTextBody,
+});
+```
+
+Notice that the object properties returned from your theme reducer function
+should map 1:1 with CSS custom property names referenced within your style
+module. In this case, `{ Color: "..." }` and `var(--Color)`.
+
+Now you should be able to reload your project and see your 'Hello World'
+component in the DOM!
+
+## Applying run time theme updates
+
+TODO
```

Comment:
This is still supposed to be TODO right?

Reply:
yea I still need to write a bit more documentation here. I will remove for now and add a JIRA card followup